### PR TITLE
Split out mopar into multiple platforms

### PR DESCRIPTION
**Pull request by rohankapoorcom**
**Description:**
The Mopar sensor has been split into multiple platforms.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21526

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/mopar.markdown [hunk 1]
```diff
@@ -8,31 +8,40 @@ comments: false
...
 logo: mopar.png
-ha_category: Car
+ha_category:
+  - Car
+  - Sensor
+  - Switch
+  - Lock
 ha_release: 0.53
 ha_iot_class: Cloud Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/mopar.markdown:18]: Only the sensor link is needed, the rest can be removed.

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/mopar.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/mopar.markdown:18]: addressed

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @rohankapoorcom! 👍

**Comment by rohankapoorcom** [comment]
Thanks for the assist @frenck!

**Pull request merged**